etc/orthography.tsv

```diff
@@ -0,0 +1,360 @@
+Grapheme	IPA	CLTS	Examples
+*	NULL	?	natriːn*
++	NULL	?	z+C:AUumgeː
+?	NULL	?	awər_?, kyjøŋ?, ɑːli_mambat͡sa?_maːʃiː
+A	?	?	z+C:AUumgeː
```

Comment:
BTW: teh first examples here could also be rendered as NULL, but I suggest the data should be corrected here.